index.html

```diff
@@ -161,10 +161,10 @@ <h2 class="cards-title">Main Program</h2>
               </p>
             </div>
           </div>
-        </div>
-        <div class="cards-bottom-line">
-          <h3 class="card-see-more">SEE THE WHOLE PROGRAM</h3>
-          <hr class="line-breaker-white" />
+          <div class="cards-bottom-line">
+            <h3 class="card-see-more">SEE THE WHOLE PROGRAM</h3>
+            <hr class="line-breaker-white" />
+          </div>
```

Comment:
- [ ] Please, as requested by the previous reviewer, you must stick to the original design as much as you can. 
- On the original design, there is no `See the whole program link`. Kindly try to hide the link on the mobile version and display the button as is shown in the original design:
Your design:
<img width="204" alt="image" src="https://user-images.githubusercontent.com/77190203/159141436-476a4664-457d-433d-a86c-1812e95c26c2.png">
Original design:
<img width="207" alt="image" src="https://user-images.githubusercontent.com/77190203/159141598-2f80431b-6d58-449c-ba59-ef6915016548.png">